bridges/modules/messages/src/lib.rs

```diff
@@ -644,6 +617,60 @@ pub mod pallet {
 		}
 	}
 
+	#[cfg(any(feature = "try-runtime", test))]
+	impl<T: Config<I>, I: 'static> Pallet<T, I> {
+		/// Ensure the correctness of the state of this pallet.
+		pub fn do_try_state() -> Result<(), sp_runtime::TryRuntimeError> {
+			Self::do_try_state_for_outbound_lanes()
+		}
+
+		/// Ensure the correctness of the state of outbound lanes.
+		pub fn do_try_state_for_outbound_lanes() -> Result<(), sp_runtime::TryRuntimeError> {
+			use sp_runtime::traits::One;
+			use sp_std::vec::Vec;
+
+			// collect unpruned lanes
+			let mut unpruned_lanes = Vec::new();
+			for (lane_id, lane_data) in OutboundLanes::<T, I>::iter() {
+				let Some(expected_last_prunned_nonce) =
+					lane_data.oldest_unpruned_nonce.checked_sub(One::one())
+				else {
+					continue;
+				};
+
+				// collect message_nonces that were supposed to be pruned
+				let mut unpruned_message_nonces = Vec::new();
+				let mut nonce_to_check = expected_last_prunned_nonce;
```

Comment:
Nit: I would rename `nonce_to_check` -> `current_nonce`  and `new_nonce_to_check` ->  `next_nonce`. Or something like that.

Reply:
fixed here: https://github.com/paritytech/polkadot-sdk/pull/4944/commits/fb8d0ba860649be82cfef18c29778ebb540b0be4